src/server/worldserver/worldserver.conf.dist

```diff
@@ -1112,6 +1112,14 @@ MaxRecruitAFriendBonusDistance = 100
 MailDeliveryDelay = 3600
 
 #
+#    CleanOldMailTime
+#        Description: The hour at which the server will be return old mails or delete them.
```

Comment:
Please remove `be` from this sentence or use `The hour at which old mails will be returned or deleted by the server.`

Reply:
Done ;)